packages/prisma/schema.prisma

```diff
@@ -1586,3 +1587,17 @@ model AttributeToUser {
 
   @@unique([memberId, attributeOptionId])
 }
+
+model Translation {
```

Comment:
Should consider adding an enum or something to point to which field on the event type this is translated for so we can translate other things in the future.

Reply:
addressed